zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/config/GatewayRestConfiguration.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.config;
+
+public class GatewayRestConfiguration {
+
+  private ProcessCacheConfiguration processCache = new ProcessCacheConfiguration();
+
+  public ProcessCacheConfiguration getProcessCache() {
+    return processCache;
+  }
+
+  public void setProcessCache(final ProcessCacheConfiguration processCacheConfiguration) {
+    processCache = processCacheConfiguration;
+  }
+
+  public static class ProcessCacheConfiguration {
+    private static final int DEFAULT_CACHE_SIZE = 100;
+
+    /**
+     * Process cache max size. Default value: {@link ProcessCacheConfiguration#DEFAULT_CACHE_SIZE}.
+     */
+    private int maxSize = DEFAULT_CACHE_SIZE;
+
+    /** Process cache expiration milliseconds. Default value: {@code null}. */
+    private Long expirationMillis = null;
```

Comment:
no caching by default?

Reply:
This only turns off the the time based eviction. Without this the cache only evicts entries when max size is reached.

We could probably find a meaningful default for this, too. I wasn't sure what could make sense and I guess it will also depend on the actual environment, so I left the default `null` (=no expiration).
Do you have any suggestion for default?

Reply:
I see, in this case, we can have up to 100 process definitions loaded in memory

Reply:
Yes, with all their flow nodes that have names.